Source/WebCore/platform/graphics/FontCascade.h

```diff
@@ -151,14 +151,15 @@ class FontCascade final : public CanMakeWeakPtr<FontCascade>, public CanMakeChec
 
     float widthOfTextRange(const TextRun&, unsigned from, unsigned to, SingleThreadWeakHashSet<const Font>* fallbackFonts = nullptr, float* outWidthBeforeRange = nullptr, float* outWidthAfterRange = nullptr) const;
     WEBCORE_EXPORT float width(const TextRun&, SingleThreadWeakHashSet<const Font>* fallbackFonts = nullptr, GlyphOverflow* = nullptr) const;
+    WEBCORE_EXPORT float width(StringView, SingleThreadWeakHashSet<const Font>* fallbackFonts = nullptr, GlyphOverflow* = nullptr) const;
```

Comment:
Since this method is never used with non-null `fallbackFonts` or `GlyphOverflow`, I wonder if we need to have our new method take these arguments?

Reply:
That's a good point. I'll get rid of it until we need it.

Reply:
Unless...do you think having it with the the extra arguments available force people to think if they need it or not?